System.Device.I2c/I2cConnectionSettings.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Device.I2c
+{
+    /// <summary>
+    /// The connection settings of a device on an I2C bus.
+    /// </summary>
+    public sealed class I2cConnectionSettings
+    {
+
+        [System.Diagnostics.DebuggerBrowsable(System.Diagnostics.DebuggerBrowsableState.Never)]
```

Comment:
why does it need the attribute?

Reply:
I'm preparing for native. So that's the reason I added here :-) same for the get => _deviceAddress
It will be required for native